modules/api/src/main/java/org/apache/ignite/lang/util/IgniteNameUtils.java

```diff
@@ -38,8 +37,6 @@ private IgniteNameUtils() {
      * @param name String to parse object name.
      * @return Unquoted name or name is cast to upper case. "tbl0" -&gt; "TBL0", "\"Tbl0\"" -&gt; "Tbl0".
      */
-    // TODO https://issues.apache.org/jira/browse/IGNITE-24021: Use QualifiedName instead.
-    @Deprecated(forRemoval = true)
     public static String parseSimpleName(String name) {
```

Comment:
This comment conflicts with
```
//  Replace this with using correct implementation of `IgniteNameUtils.parseSimpleName`, when it will be fixed.
String schemaNameInCanonicalForm = QualifiedName.fromSimple(defaultSchemaName).objectName();
```
Also 
```
QualifiedName.fromSimple(defaultSchemaName).objectName();
```
looks confusing, considering that we are passing the name of the scheme

Reply:
Let's rename method to  `parseIdentifier`

Reply:
renamed